packages/@react-spectrum/s2/src/ContextualHelp.tsx

```diff
@@ -41,8 +41,8 @@ export interface ContextualHelpProps extends
 
 const popover = style({
   fontFamily: 'sans',
-  minWidth: 218,
-  width: 218,
+  minWidth: 269,
+  width: 269,
```

Comment:
268?

Reply:
I was trying to trigger the build comment. Updated